examples/arkode/C_parallel/ark_brusselator1D_task_local_nls_--monitor_--explicit_--tf_3.out

```diff
@@ -24,47 +24,47 @@ Output directory: .
-       1.875000     1.529809     2.264525     3.499975
-       1.950000     1.477846     2.274002     3.499976
-       2.025000     1.423688     2.291477     3.499972
-       2.100000     1.368027     2.316515     3.499979
-       2.175000     1.311469     2.348594     3.499982
-       2.250000     1.254550     2.387108     3.499969
-       2.325000     1.197744     2.431383     3.499975
-       2.400000     1.141466     2.480692     3.499993
-       2.475000     1.086075     2.534288     3.499986
-       2.550000     1.031874     2.591422     3.499982
-       2.625000     0.979114     2.651378     3.499982
-       2.700000     0.927998     2.713489     3.499987
-       2.775000     0.878688     2.777156     3.499983
-       2.850000     0.831306     2.841860     3.499983
-       2.925000     0.785951     2.907162     3.499994
-       3.000000     0.742699     2.972704     3.499991
+       0.750000     1.421770     3.076177     3.499965
+       0.825000     1.481479     2.993308     3.499971
+       0.900000     1.541887     2.906731     3.499979
+       0.975000     1.599823     2.818833     3.499973
+       1.050000     1.651718     2.732272     3.499979
+       1.125000     1.694285     2.649571     3.499975
+       1.200000     1.725150     2.572793     3.499976
+       1.275000     1.743172     2.503400     3.499982
+       1.350000     1.748376     2.442296     3.499973
+       1.425000     1.741639     2.389972     3.499970
+       1.500000     1.724315     2.346645     3.499973
+       1.575000     1.697930     2.312381     3.499968
+       1.650000     1.663986     2.287150     3.499964
+       1.725000     1.623866     2.270867     3.499968
+       1.800000     1.578787     2.263391     3.499967
+       1.875000     1.529809     2.264525     3.499966
+       1.950000     1.477846     2.274002     3.499973
+       2.025000     1.423688     2.291477     3.499976
+       2.100000     1.368027     2.316515     3.499976
+       2.175000     1.311469     2.348594     3.499990
+       2.250000     1.254550     2.387108     3.499986
+       2.325000     1.197744     2.431383     3.499983
+       2.400000     1.141466     2.480692     3.499988
+       2.475000     1.086075     2.534288     3.499985
+       2.550000     1.031874     2.591422     3.499970
+       2.625000     0.979114     2.651378     3.499976
+       2.700000     0.927998     2.713489     3.499977
+       2.775000     0.878688     2.777156     3.499974
+       2.850000     0.831306     2.841860     3.499986
+       2.925000     0.785951     2.907162     3.499997
+       3.000000     0.742699     2.972704     3.499994
 
 Final Solver Statistics (for processor 0):
-   Internal solver steps = 238744 (attempted = 239406)
-   Total RHS evals:  Fe = 718221   Total number of error test failures = 662
+   Internal solver steps = 236112 (attempted = 332018)
+   Total RHS evals:  Fe = 996057   Total number of error test failures = 95906
```

Comment:
Steven confirmed this increase in the number of error test failures is fixed by the updated controller defaults in PR #565

Reply:
Fixed by commit 40dd2bb